plugins/gcalendar-homepage/src/icons/calendarIcon.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="186 38 76 76">
```

Comment:
A general question on these icons - are they trademarked or similarly legally sensitive? Is it known to be licensed in such a way that they are OK to add to an open source project?

Reply:
Took this from https://commons.wikimedia.org/wiki/File:Google_Calendar_icon_(2020).svg, don't think that can be a problem.. but we can use some MaterialUI icon by default and allow to pass icon as a prop

Reply:
Oh wikimedia commons should be an OK source afaik